cgmanifests/cgmanifest.json

```diff
@@ -19,7 +19,7 @@
       {
         "component": {
           "Type": "maven",
-          "maven": { "GroupId": "com.google.protobuf", "ArtifactId": "protobuf-java", "Version": "3.20.1" },
```

Comment:
Then you will re-introduce the problem that was fixed in https://github.com/microsoft/onnxruntime/pull/10420 See: https://github.com/advisories/GHSA-wrvw-hg22-4m67